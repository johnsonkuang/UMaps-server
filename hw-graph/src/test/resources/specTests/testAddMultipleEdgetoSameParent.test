# Test Add Multiple Edge to Same Parent
# Tests whether a parent node can support multiple connections to different children

# Create a Graph
CreateGraph graph1

# Add parent and children nodes
AddNode graph1 parent
AddNode graph1 child1
AddNode graph1 child2
AddNode graph1 child3
AddNode graph1 child4
AddNode graph1 child5

# Add edges connecting parent and children
AddEdge graph1 parent child1 A
AddEdge graph1 parent child2 B
AddEdge graph1 parent child3 C
AddEdge graph1 parent child4 D
AddEdge graph1 parent child5 E