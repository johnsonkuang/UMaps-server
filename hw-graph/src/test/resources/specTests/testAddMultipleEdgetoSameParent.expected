# Test Add Multiple Edge to Same Parent
# Tests whether a parent node can support multiple connections to different children

# Create a Graph
created graph graph1

# Add parent and children nodes
added node parent to graph1
added node child1 to graph1
added node child2 to graph1
added node child3 to graph1
added node child4 to graph1
added node child5 to graph1

# Add edges connecting parent and children
added edge A from parent to child1 in graph1
added edge B from parent to child2 in graph1
added edge C from parent to child3 in graph1
added edge D from parent to child4 in graph1
added edge E from parent to child5 in graph1